src/github.ts

```diff
@@ -1477,7 +1477,13 @@ export class GitHub {
       previous_tag_name: previousTag,
       target_commitish: targetCommitish,
     });
-    return resp.data.body;
+
+    const versionWithoutV = resp.data.name.replace(/^v/, '');
+
+    return resp.data.body.replace(
```

Comment:
I'm open to any kind of combination of version header & body, The one with just a version like in the default strategy and a list of changes looks like the most concise to me, though I didn't make the version header a link, because GitHub already adds the compare changes link to the bottom